pom.xml

```diff
@@ -124,5 +124,33 @@
         </plugins>
     </reporting>
 
+    <profiles>
+        <profile>
+            <id>openrewrite</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.openrewrite.maven</groupId>
+                        <artifactId>rewrite-maven-plugin</artifactId>
+                        <version>5.23.1</version>
+                        <configuration>
+                            <activeRecipes>
+                                <recipe>org.openrewrite.java.OrderImports</recipe>
```

Comment:
This one has not been applied yet; I do feel it would be beneficial to have a standard lightly enforced in the manner though, as IDEAs and settings will change between users, and automated enforcement then stops all discussions.